plugins/slick.autotooltips.ts

```diff
@@ -5,55 +5,61 @@ type GenericOption = Record<string, Primitive>;
 // @todo slickGrid type should be in some generic type folder
 type SlickGrid = any;
 
-
```

Comment:
Did you actually make some changes? Hard to tell with all the formatting changes

Reply:
nvm, I think see the changes. Thanks for fixing ;)